[DM-2111] lwm2m improvement - required availability (#2305)
SoftwareAG/c8y-docs: content/protocol-integration/lwm2m-bundle/registering-lwm2m-devices.md

<td style="text-align: left">Optional</td>
</tr>
<tr>

> <td style="text-align: left">Enable/Disable automatic management of availability required interval</td>

Reviewer: "automatic management of availability required interval": Is this really one term? This is quite long and hard to understand. Could we split it somehow? For example: Interval for automatic availability management? What is it called in the UI? I don´t think, that "required" is needed here? @Oezge-Akin can you please check the wording?

Reviewer 2: It is related to required interval field
https://cumulocity.com/docs/device-management-application/monitoring-and-controlling-devices/#to-monitor-the-connection-of-a-particular-device

Is this wording better "Automatic setting of required interval" to make it shorter and in tooltip we can add the details.

Reviewer: Yes, much clearer. Please change in the release notes title

Author: done

<td style="text-align: left">autoManageAvailabilityRequiredInterval</td>
<td style="text-align: left">Boolean</td>

> <td style="text-align: left">When true (default), LWM2M agent will automatically set required interval to registration lifetime plus 2 minutes.</td>

Reviewer: ```suggestion
<td style="text-align: left">When true (default), the LWM2M agent automatically sets the interval to registration lifetime plus 2 minutes.</td>
```
